community-membership.md

```diff
@@ -249,3 +250,17 @@ it's incredibly important for the project to maintain velocity and recognize
 that we are all coming to these sorts of technical issues with the best of
 intentions and remain aligned about the overall goals of the OpenTelemetry
 project.
+
+## Emeritus Maintainer/Approver/Triager
+
+For any healthy open source project, it is inevitable that maintainers may move
```

Comment:
nit: we should use a word other than "maintainer" here since it's not just for maintainers :)

Reply:
7b1ff03946aaeeffad4d56e0bd53796dbc32d143